.github/workflows/conda.yml

```diff
@@ -28,9 +28,22 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: true
-          fetch-depth: 0  # history required so cmake can determine version
+          fetch-depth: 0 # history required so cmake can determine version
 
       - uses: conda-incubator/setup-miniconda@v3
+      - name: Create variant file for conda build
+        run: |
+          echo "python:" > dynamic_var.yaml
+          if [ "${{ inputs.python-version }}" == "3.13" ]; then
+            echo "- 3.13.* *_cp313" >> dynamic_var.yaml
+          else
+            echo "- ${{ inputs.python-version }}.* *_cpython" >> dynamic_var.yaml
+          fi
+          echo "numpy:" >> dynamic_var.yaml
+          echo "- '2'" >> dynamic_var.yaml
+          echo "python_impl:" >> dynamic_var.yaml
+          echo "- cpython" >> dynamic_var.yaml
```

Comment:
We need to generate this config on the fly as conda-build CLI can't merge the python config for python 3.13 from the CLI argument `--python`. conda-build doesn't support python 3.13 yet.